captum/optim/_core/loss.py

```diff
@@ -237,28 +429,65 @@ def __call__(self, targets_to_values: ModuleOutputMapping) -> torch.Tensor:
                     (activ_a - activ_b) ** 2
                 ).mean() / self.decay_ratio ** float(d)
 
-        return sum_tensor
+        return -sum_tensor
 
 
-class Direction(Loss):
+def _dot_cossim(

```

Comment:
We might want to put `_dot_cossim` in `captum.optim._utils.image.common` like Narine had me do with `get_neuron_pos`.